### PR TITLE
Replace the graphic on the Layers reference page

### PR DESCRIPTION
**Pull request by illright**
## Background

The Layers reference page doesn't tell anything about the fact that layers are just folders. This updated graphic is meant to reflect the file-system nature of layers.

## Changelog

1. Changed the graphic to the new one
2. Added a descriptive alt text to the graphic
3. Added styles to support themed images with `#light-mode-only` and `#dark-mode-only` URL modifiers

Before:
telegram-cloud-photo-size-2-5431876295400215458-y

After:
image

image

### CODE REVIEW

File: i18n/en/docusaurus-plugin-content-docs/current/reference/layers.mdx [hunk 1]
```diff
@@ -15,9 +15,10 @@ On this page, a _module_ refers to an internal module in the application — a f
 
 Every layer carries special semantic meaning to help you determine how much responsibility you should allocate to a module in your code. The names and meanings of layers are standardized across all projects built with Feature-Sliced Design.
 
-There are 7 layers in total, arranged from most responsibility and dependency to least:
+There are **7 layers** in total, arranged from most responsibility and&nbsp;dependency to least:
```
Comment (azinit) [inline i18n/en/docusaurus-plugin-content-docs/current/reference/layers.mdx:18]: nitpick: Добавь еще в рус версию пож

**Review by azinit — CHANGES_REQUESTED** [review]

File: i18n/en/docusaurus-plugin-content-docs/current/reference/layers.mdx (hunk 1 above)
Comment (illright) [reply i18n/en/docusaurus-plugin-content-docs/current/reference/layers.mdx:18]: а, блин, забыл)

**Review by azinit — APPROVED** [review]
ready for merge 🔥

**Comment by azinit** [comment]
@illright could u please merge `master` into `current` branch after recent changes? 🙏

**Pull request merged**